jb/itch5/compute_book_depth.hpp

```diff
@@ -0,0 +1,50 @@
+#ifndef jb_itch5_compute_book_depth_hpp
+#define jb_itch5_compute_book_depth_hpp
+
+#include <jb/itch5/compute_base.hpp>
+
+namespace jb {
+namespace itch5 {
+
+/** 
+ * An implementation derivated fomr compute_base to callback the book depth.
+ */
+
+class compute_book_depth : public compute_base {
```

Comment:
Please consider a private base class here.  This is not a IS-A relationship, it is simply a case of sharing implementation.